### ISSUE TITLE
Request: Have option for level 1 mobs to be easier than vanilla

### ISSUE THREAD

**Issue by stumper66** [lokka30/LevelledMobs#227]
suggested by `_Luke_#0001`

**Is your feature request related to a problem? Please describe.**
I'd like to have an option for level 1 mobs to be easier than vanilla mobs

**Describe the solution you'd like**
Level 1 mobs move slower, have less health and do less damage than vanilla mobs

**Comment by lokka30** [comment]
Is this not possible with LM already? I am thinking that perhaps a attrib multiplier less than 1 (0.9, 0.5, 0.3 etc) could do the trick, but I am not sure.

**Comment by UltimaOath** [comment]
Tested and you can use a negative value between -1.0 and -0.01 , though hp
can never be less than one to prevent instant death.

**Comment by UltimaOath** [comment]
Example:

      max-health: -0.5
      
That means max health will apply at -50%.

**Comment by stumper66** [comment]
So the problem is want level 1 to be easier than vanilla but higher levels basically normal.
I believe level 1s don't get any attribute modifiers applied

**Comment by lokka30** [comment]
> 
> 
> So the problem is want level 1 to be easier than vanilla but higher levels basically normal.
> I believe level 1s don't get any attribute modifiers applied

Ahh, I see what you mean now. Thanks for the clarification. Wouldn't this change the formula?

**Comment by UltimaOath** [comment]
You are correct after my testing. I threw together what might be a new system for LM if the idea is sound, easy enough to implement, and desired.
```
    multipliers:
      tiered:
        1-9:
          max-health: 1.0
          movement-speed: 1.0
          attack-damage: 1.0
          ranged-attack-damage: 1.0
          item-drop: 1
          xp-drop: 1
        10-19:
          max-health: 2.0
          movement-speed: 2.0
        15:
          custom-mob-level:
            ZOMBIE:
              xp-drop: 3
```
Demonstrated above, adding a 'tiered:' mechanic to the multipliers which will allow you to establish different multipliers for different level ranges to utilize. It would function by using the tiered values in place of the minLevel/maxLevel utilized in the formula for calculating the attributes.
For example, under the first tiered: of `1-9`, if an entity would have spawned at any level between minLevel 1 and maxLevel 9 under established conditions would use the listed multipliers rather than any default established for that range. So LVL 1 would be 'vanilla', while LVL 9 would be 100% increase (based on demo), with all levels in the middle some portion n between.

Then you have the next set of tiered: , called `10-19`. Under this heading, only two multipliers are listed: max-health and movement-speed. That would mean any entity which spawned at any level between LVL 10 through LVL 19 would use the altered movement-speed and max-health multipliers, but then refer to default rule for other values missing. This would process out in the same fashion as the above: where LVL 10 would be vanilla, while LVL 19 would be 200% increase specifically for the max-health and movement-speed (based on demo), with all levels in the middle a portion between, while the other attributes would scale however the default rule established.

Finally you have single number overrides, demonstrated at the end. With a single number listed, it's counted as both the minLevel and maxLevel when calculating things, and would alter the setting for that one specific level, overriding should that value sit within a range already established elsewhere. This override could potentially be a setting to include whether it overrides or no.t. As for the demonstration, at LVL 15, I specify that zombies would have an increased XP rate compared to default rule.

Long story short I believe something like this might be a potential solution to the issue at hand, demonstrated basically below:
```
    multipliers:
      tiered:
        1:
          max-health: -0.5
          movement-speed: -0.5
          attack-damage: -0.5
          ranged-attack-damage: -0.5
          item-drop: 0
          xp-drop: 0
```

**Comment by UltimaOath** [comment]
And all we'd have to do regarding the lvl 1 entities not getting levels I believe would be to remove the -1 from the formula if I'm not mistaken.

**Comment by lokka30** [comment]
I feel like this has a niche use case. Is it worth having the medium priority, or should we reconsider?

**Comment by lokka30** [comment]
This issue has not had activity for almost a month, and I still believe this is a very niche issue which can be solved by tinkering with rules system values. LM is meant to make mobs harder, not easier.

Closing this issue, re-open if you want

**Comment by Temyxxa** [comment]
> Вы правы после моего тестирования. Я собрал то, что могло бы стать новой системой для LM, если бы идея была разумной, достаточно простой в реализации и желаемой.
> 
> ```
>     multipliers:
>       tiered:
>         1-9:
>           max-health: 1.0
>           movement-speed: 1.0
>           attack-damage: 1.0
>           ranged-attack-damage: 1.0
>           item-drop: 1
>           xp-drop: 1
>         10-19:
>           max-health: 2.0
>           movement-speed: 2.0
>         15:
>           custom-mob-level:
>             ZOMBIE:
>               xp-drop: 3
> ```
> 
> Как показано выше, добавление многоуровневой механики к множителям, которая позволит вам установить разные множители для разных диапазонов уровней для использования. Он будет работать, используя многоуровневые значения вместо minLevel/maxLevel, используемых в формуле для вычисления атрибутов. Например, в первом уровне: of `1-9`, если бы объект появлялся на любом уровне между minLevel 1 и maxLevel 9 при установленных условиях, он использовал бы перечисленные множители, а не какие-либо значения по умолчанию, установленные для этого диапазона. Таким образом, LVL 1 будет «ванильным», а LVL 9 будет увеличен на 100% (на основе демонстрации), со всеми уровнями посередине с некоторой частью n между ними.
> 
> Затем у вас есть следующий набор многоуровневых: , называемый `10-19`. Под этим заголовком указаны только два множителя: максимальное здоровье и скорость передвижения. Это будет означать, что любая сущность, появившаяся на любом уровне между LVL 10 и LVL 19, будет использовать измененные множители скорости движения и максимального здоровья, но затем ссылаться на правило по умолчанию для других отсутствующих значений. Это будет обрабатываться так же, как и выше: где LVL 10 будет ванильным, а LVL 19 будет увеличено на 200% специально для максимального здоровья и скорости движения (на основе демонстрации), со всеми уровнями посередине часть между ними, в то время как другие атрибуты будут масштабироваться в соответствии с установленным правилом по умолчанию.
> 
> Наконец, у вас есть переопределение одного числа, продемонстрированное в конце. С одним указанным числом оно считается как minLevel, так и maxLevel при расчете вещей и изменит настройку для этого конкретного уровня, переопределяя, если это значение находится в пределах диапазона, уже установленного в другом месте. Это переопределение потенциально может быть настройкой, включающей, переопределяет ли оно или нет. Что касается демонстрации, то на 15 лвл я указываю, что у зомби будет повышенный показатель опыта по сравнению с правилом по умолчанию.
> 
> Короче говоря, я считаю, что что-то вроде этого может быть потенциальным решением проблемы, продемонстрированной в основном ниже:
> 
> ```
>     multipliers:
>       tiered:
>         1:
>           max-health: -0.5
>           movement-speed: -0.5
>           attack-damage: -0.5
>           ranged-attack-damage: -0.5
>           item-drop: 0
>           xp-drop: 0
> ```

Greetings!
This advice doesn't work for me.
Tried values -0.1, -0.8, -1.1, -20, -100
No effect
